test/unit/helpers/dynamic_table_helper_test.rb

```diff
@@ -48,6 +48,15 @@ class DynamicTableHelperTest < ActionView::TestCase
       assert_equal columns_count, dt[:columns].length
       dt[:rows].each { |r| assert_equal columns_count, r.length }
 
+      unless (dt[:rows][0].any? { |x| x == '' }) || (dt[:rows][1].any? { |x| x == '' }) || (dt[:rows][2].any? { |x| x == '' })
+        puts
+        puts "Flaky test debug:"
+        puts
+        puts dt.inspect
+        puts
+        pp study.sample_types
+      end
+
```

Comment:
some debugging puts code

Reply:
I was going to leave that in to try and figure out why that test randomly fails sometimes